demos/ASP.Net Core/Divergent.Frontend/Controllers/HomeController.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace Divergent.Frontend.Controllers
+{
+    public class HomeController : Controller
+    {
+        public IActionResult Index()
+        {
+            return RedirectToRoute(new { controller = "Orders" });
+        }
```

Comment:
![image](https://user-images.githubusercontent.com/677704/28464982-2631c846-6e1f-11e7-8dc2-74c3e819ca2e.png)

Fair enough. Let's leave as-is. The VS templates should really be updated.